### PR TITLE
Add input for a buildah cheat sheet

### PR DESCRIPTION
**Pull request by matthiasclasen**

### CODE REVIEW

File: ideas/buildah-cheat-sheet [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# buildah rm CONTAINER_ID
+
+Mount a working container filesystem:
+# buildah mount CONTAINER_ID
+
+Unmount a working container filesystem:
+# buildah umount CONTAINER_ID
+
+Run a working container:
+# buildah run CONTAINER_ID COMMAND
```
Comment (miabbott) [inline ideas/buildah-cheat-sheet]: The changes in projectatomic/buildah#734 suggest that `buildah run` is the equivalent of the `RUN` command in a Dockerfile.  Whereas `podman run` is to be used for actually running the container itself.

We should probably clarify this here for the users.

File: ideas/buildah-cheat-sheet [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+--net "container"|"host"  for network access
+--pid "container"|"host"  for pid namespace
+
+Create a working container from a dockerfile:
+# buildah bud -f FILE CONTEXT
+
+Example:
+# buildah bud -f Dockerfile.simple .
+
+Enumerate existing containers:
```
Comment (miabbott) [inline ideas/buildah-cheat-sheet]: I'm not sure if we want to steer users to using `buildah` for these kind of tasks.  I think `buildah` is best suited for the actual building of containers, whereas `podman` is more appropriate for the management, running, removing, etc type of tasks.

Comment (matthiasclasen) [reply ideas/buildah-cheat-sheet]: well, its a buildah cheat sheet...

**Comment by miabbott** [comment]
This is a good start and I'd like to see it expanded to show more examples of building containers without the use of a Dockerfile.

For example, I've migrated a Dockerfile to a bash script using `buildah` here - https://github.com/miabbott/files/blob/master/dockerfiles/myprecious/buildah.sh

File: ideas/buildah-cheat-sheet (hunk 2 above)
Comment (miabbott) [reply ideas/buildah-cheat-sheet]: Fair.  We can keep this and maybe add a note about using `podman` for most container mgmt tasks?

Comment (matthiasclasen) [reply ideas/buildah-cheat-sheet]: how does this work, anyway ? if I create a container with buildah, does podman see it ? are they using the same namespace for naming containers ?

Comment (miabbott) [reply ideas/buildah-cheat-sheet]: > if I create a container with buildah, does podman see it ?

Yup, they both use `/var/run/containers/storage`, whereas `docker` uses `/var/lib/docker`

The caveat is, if you build an image with `buildah`, `docker` won't be able to enumerate it.  And conversely, images built with `docker` will not be enumerated by `buildah`/`podman`.

**Pull request merged**